### PR TITLE
[alpine-curl] Alpine-based curl image

### PR DESCRIPTION
**Pull request by masaruhoshi**

### CODE REVIEW

**Comment by qlikossbuild** [comment]
Hi, and thanks for contributing!

Note: If you are part of Qlik R&D, you can ignore this process (but let us know in a comment below!)

Before we can handle your pull request we need you to sign our contributor license agreement.
This is a one-time thing and enables you to contribute without hassle to any of the qlik-oss repositories in the future.

And it only takes a few minutes! Here are some useful links to get you started:

* [Sign the CLA](https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sign-the-cla)
* [Rationale behind the CLA](https://github.com/qlik-oss/open-source/blob/master/cla-rationale.md)
* [Qlik R&D Open Source repository](https://github.com/qlik-oss/open-source#open-source-at-qlik-rd)
* [Official Slack team](http://branch.qlik.com/slack)

Having troubles? Feel free to reach out to us either here or on Slack!

File: curl/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.9.3
+MAINTAINER Masaru Hoshi <hoshi@hoshi.com.br>
```
Comment (hairyhenderson) [inline curl/Dockerfile]: the `MAINTAINER` instruction is deprecated! use `LABEL maintainter=` instead

File: curl/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+RUN apk add --update --no-cache curl && \
+    rm -fr /var/cache/apk/*
```
Comment (hairyhenderson) [inline curl/Dockerfile]: the `rm` should be unnecessary if you use `--no-cache`

File: curl/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.9.3
+MAINTAINER Masaru Hoshi <hoshi@hoshi.com.br>
+
+RUN apk add --update --no-cache curl && \
```
Comment (hairyhenderson) [inline curl/Dockerfile]:
```suggestion
RUN apk add --no-cache curl
```

File: curl/readme.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+    The Docker image can be built using the following command
+
+    ```{bash}
+    docker build -t <your Docker Hub ID>/curl .
+    ```
+
+    where the dot is the current folder, assuming you are running the docker command from the same folder of the Dockerfile.
+
+    ## License
```
Comment (hairyhenderson) [inline curl/readme.yaml]: I don't think this section is necessary (it's covered by the `Third-Party Licenses` boilerplate that gets templated in)

**Pull request closed without merging**